web-client/integration-tests-public/journey/unauthedUserSearchesByMeta.ts

```diff
@@ -32,7 +32,7 @@ export const unauthedUserSearchesByMeta = (cerebralTest, overrides = {}) => {
       cerebralTest.getState(`searchResults.${ADVANCED_SEARCH_TABS.CASE}`),
     ).toBeUndefined();
     expect(cerebralTest.getState('advancedSearchForm')).toEqual({
-      caseSearchByName: { countryType: ALL_COUNTRY_TYPE },
+      caseSearchByName: { countryType: ALL_COUNTRY_TYPE, procedureType: 'All' },
```

Comment:
I wonder if it is worth having a more generic "all options" constant. This feels a little clunky given that we use both a constant and a non-constant.